docs/content/docs/start/react-tutorial.md

```diff
@@ -0,0 +1,295 @@
+---
+title: 'Tutorial: Create a Fluid Framework application with React'
+menuPosition: 4
+---
+
+In this tutorial, you'll learn about using the Fluid Framework by building a simple application that enables every client of the application to change a dynamic time stamp on itself and all other clients almost instantly. You'll also learn how to connect the Fluid data layer with a view layer made in [React](https://reactjs.org/). The following image shows the time stamp application open in four browsers. Each has a button labelled **click** and beside it a UNIX Epoch time. The same time in all four. The cursor is on the button in one browser.
+
+![Four browsers with the Timestamp app open in them.](images/Four-clients-1.PNG)
+
+The following image shows the same four clients one second after the **click** button was pressed. Note that the timestamp has updated to the very same time in all four browsers.
+
+
+![Four browsers with the Timestamp app open in them one second after the button has been pushed.](images/Four-clients-2.PNG)
+
+{{< callout note >}}
+
+This tutorial assumes that you are familiar with the [Fluid Framework Overview](../overview.md) and that you have completed the [QuickStart](./quick-start.md). You should also be familiar with the basics of [React](https://reactjs.org/), [creating React projects](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app), and [React Hooks](https://reactjs.org/docs/hooks-intro.html).
+
+{{< /callout >}}
```

Comment:
@tylerbutler , to confirm the proper syntax here. I think it's this: 

```suggestion
{{% callout note %}}

This tutorial assumes that you are familiar with the [Fluid Framework Overview](../overview.md) and that you have completed the [QuickStart](./quick-start.md). You should also be familiar with the basics of [React](https://reactjs.org/), [creating React projects](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app), and [React Hooks](https://reactjs.org/docs/hooks-intro.html).

{{% /callout %}}
```

Reply:
Both work in most cases.